cypress/fixtures/evidence_requests/index.json

```diff
@@ -39,7 +40,8 @@
       }
     ],
     "team": "Development Housing Team",
-    "reason": "staging-reason"
+    "reason": "staging-reason",
+    "userRequestedBy": "test@email"
```

Comment:
Small thing but in other fixtures we used `test@hackney.gov.uk`. Could use the same for consistency

Reply:
Sure, I'll change it to that